metricbeat/module/kibana/stats/_meta/data.json

```diff
@@ -6,64 +6,54 @@
     },
     "elasticsearch": {
         "cluster": {
-            "id": "PKEQ1V5kT4yPng_sgiqF9g"
+            "id": "NkfU5AinRnyFnqBD36zhEw"
         }
     },
     "kibana": {
         "stats": {
-            "concurrent_connections": 17,
-            "event_loop_delay": 718.5184001922607,
-            "name": "kibana",
+            "concurrent_connections": 3,
+            "kibana": {
```

Comment:
What if we put this data up one level outside `stats`?

So it would be for example `kibana.host.name`, `kibana.name`, `kibana.id` etc.? I would assume in case we add other metricsets they would reuse this data.

It bugs me that we have `kibana.*.kibana`.